packages/turf-truncate/index.js

```diff
@@ -0,0 +1,55 @@
+const deepSlice = require('deep-slice');
```

Comment:
This file is using `const` and for of - so far, Turf is ES5 and this addition would break IE11 support. Could this stick to ES5 syntax?